content/docs/lists-and-keys.md

```diff
@@ -202,13 +201,13 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
+[**CodePen पर इसे आजमाएं**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
 
-A good rule of thumb is that elements inside the `map()` call need keys.
+एक अच्छा नियम यह है कि एलिमेंटस  को `map()` कॉल के समय पर key  की जरुरत पड़ती है।  
 
-### Keys Must Only Be Unique Among Siblings {#keys-must-only-be-unique-among-siblings}
+### keys सिर्फ़ भाई-बहनों के बीच अद्वितीय होनी चाहिए{#Keys-Must-Only-Be-Unique-Among-Siblings}
```

Comment:
`keys` => `Keys`

Reply:
`भाई-बहनों` => `सिब्लिंग्स`

Reply:
`चाहिए{#Keys-Must-Only-Be-Unique-Among-Siblings}` => `चाहिए {#keys-must-only-be-unique-among-siblings}`